Automattic/wp-calypso
# Datacenter Picker: Use the correct spelling for 'data center' throughout

From https://github.com/Automattic/wp-calypso/pull/70237

See https://github.com/Automattic/dotcom-forge/issues/1237

## Proposed Changes

Uses the correct spelling for 'data center' throughout.

![image](https://user-images.githubusercontent.com/36432/204615128-30ec8e36-4a90-4f4d-8cb6-add30bc60f91.png)

![image](https://user-images.githubusercontent.com/36432/204615119-093469a2-2189-4a27-849a-c5a94e7e3911.png)

## Testing Instructions

1. Create a new WordPress.com site with a business plan.
2. Navigate to 'Hosting Configuration' and click 'Activate'.
3. Verify the `<EligibilityWarnings>` component displays with the Data center Picker UI.
4. Verify the Data center Picker UI is usable, accessible, and looks good in the browsers we support.
5. If you want to recreate the 'no eligibility warnings' state, map a domain to your site and then go back to 'Hosting Configuration' -> 'Activate'.
6. Pick a custom datacenter, click 'Continue', and verify the site is created in the expected datacenter.
7. Navigate to 'Hosting Configuration' after activation and verify the selected data center is presented.
